pygeo/mphys/mphys_dvgeo.py

```diff
@@ -141,21 +141,23 @@ def nom_add_point_dict(self, point_dict):
         for k, v in point_dict.items():
             self.nom_addPointSet(v, k)
 
-    def nom_addGlobalDV(self, dvName, value, func, childIdx=None):
+    def nom_addGlobalDV(self, dvName, value, func, childIdx=None, add_input=True):
         # global DVs are only added to FFD-based DVGeo objects
         if self.geo_type != "ffd":
             raise RuntimeError(f"Only FFD-based DVGeo objects can use global DVs, not type:{self.geo_type}")
 
         # define the input
-        self.add_input(dvName, distributed=False, shape=len(value))
+        # When composite DVs are used, input is not required for the default DVs
+        if add_input:
```

Comment:
I still think this is unclear when you just read the code and don't have the context of the PR. Because `add_input` is used IFF the DV is composite (correct me if I'm wrong), I think it would make more sense to have the variable name reflect that. Like if the variable name were `isComposite` or something similar so the check becomes 
`if isComposite():
      <do thing`
and then the comment can say that the initial setup of the composite DV makes the extra step necessary.